pkg/apis/componentconfig/types_pluginargs.go

```diff
@@ -131,3 +132,17 @@ type HighNodeUtilizationArgs struct {
 	Thresholds    api.ResourceThresholds
 	NumberOfNodes int
 }
+
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// IsEvictableFilterArgs holds arguments used to configure IsEvictableFilter plugin.
+type IsEvictableFilterArgs struct {
```

Comment:
s/IsEvictableFilterArgs/DefaultEvictorArgs

Reply:
You mean DefaultEvictorFilterArgs? Will we have different args for each filter plugin and each evictor plugin, right?

Reply:
I think my question is, will eviction filters be independent of evictors?